keras/src/backend/tensorflow/image.py

```diff
@@ -15,6 +16,18 @@
 )
 
 
+def psnr(image1, image2, max_val):
+    if image1.shape != image2.shape:
+        raise ValueError(
+            f"Image shapes {image1.shape} and {image2.shape} must match for PSNR calculation."
+        )
+
+    max_val = convert_to_tensor(max_val, dtype=image1.dtype)
+    mse = tf.reduce_mean(tf.square(image1 - image2), axis=[-3, -2, -1])
+    psnr_val = 20 * tfnp.log10(max_val) - 10 * tfnp.log10(mse)
```

Comment:
Hi @IMvision12 
I'm trying to remove the dependency on `tfnp` in #19556 

I think you could use something like this in the `psnr` function to prevent future breakage, and it should be more consistent regarding dtype promotion.
```python
from keras.src.backend.tensorflow.numpy import log10
```

Reply:
@james77777778 Ok, That's cool!

Reply:
@james77777778 any idea on failing tests?